server/src/services/auth.service.ts

```diff
@@ -66,7 +72,16 @@ export class AuthService {
 
     const { email } = params;
     const organization = await this.organizationsService.create('Untitled organization');
-    const user = await this.usersService.create({ email }, organization, ['all_users', 'admin']);
+    let user;
+    try {
+      user = await this.usersService.create({ email }, organization, ['all_users', 'admin']);
```

Comment:
I think it might be better to check for existence of an email before trying to create a new user and then throw an error than relying on error code.

Reply:
@akshaysasidrn wdyut?

Reply:
> I think it might be better to check for existence of an email before trying to create a new user and then throw an error than relying on error code.

Actually, this makes more sense. I have patched this in the latest commit.